dev/io.openliberty.org.apache.jasper.expressionLanguage.5.0_fat/test-applications/EL50BasicTests.war/src/io/openliberty/el50/fat/basic/servlets/EL50BasicTestsServlet.java

```diff
@@ -0,0 +1,166 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.el50.fat.basic.servlets;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import jakarta.el.ELContext;
+import jakarta.el.ELManager;
+import jakarta.el.ELProcessor;
+import jakarta.el.ELResolver;
+import jakarta.el.ExpressionFactory;
+import jakarta.el.StandardELContext;
+import jakarta.el.ValueExpression;
+import jakarta.servlet.annotation.WebServlet;
+
+
+/**
+ * Servlet for testing basic changes in EL 5.0
+ */
+@WebServlet({ "/EL50BasicTests" })
+public class EL50BasicTestsServlet extends FATServlet {
```

Comment:
Any reason to not set the class to FULL mode rather than each method? Looks like both are in FULL mode right now.

Reply:
Main reason was the intention of adding more tests in this class as I work through more of the issues but I can switch it to full for these changes and update as I deliver more.

Reply:
I'm fine either way, if you have plans to add more tests here you can leave it as is and we can always clean it up once all the tests are delivered and we do a final test review.